mood/distance.py

```diff
@@ -12,6 +13,8 @@ def get_distance_metric(example):
     metric = "euclidean"
 
     # For binary vectors we use jaccard
+    if isinstance(example, pd.DataFrame):
+        example = example.values  # DataFrames would require all().all() otherwise
```

Comment:
This might get slow with large feature vectors or large datasets. Maybe: 

```suggestion
        example = example.head(10).values  # DataFrames would require all().all() otherwise
```

Reply:
Hi @cwognum, thanks for your feedback and for considering my contribution! I've been using your package so I appreciate this opportunity. Indeed, I think in the beginning scikit-learn wasn't supporting Pandas DataFrames as input, but the compatibility has increased over their releases.

I like your suggestion but maybe this could also be done whenever `example` is also a `np.array()`? My latest commit added the line:
`example = example[:10, :] if (example.ndim == 2 and example.shape[0] > 10) else example`

Again, already formatted with `black`! Let me know your thoughts

Reply:
Actually - Sorry, but let's revert it to how you had it originally. I think it's good to leave this decision up to the user.

Let me make these latest changes and merge this PR! 

Thank you for thinking along and making a contribution!

Reply:
Done! Sorry for the extra commits, I accidentally reverted the original change by mistake...